### PR TITLE
Tweaking email language re. repertory

### PR DESCRIPTION
**Pull request by millsjdm**

### CODE REVIEW

**Comment by dbinetti** [comment]
So a few things:

I'm really not sure the competitor is going to appreciate the nuance between repertory and other changes.  But more important: the Package email is not correct.  Once the session is packaged, it's done.  The legacy report will not update to reflect repertory even if it changes in the interim; in this case, FINAL really does mean final.

Now what you choose to do with that policy-wise is another thing altogether, but basically if you want them to get a song updated after packaging then they should do so by looking up the group repertory in Barberscore (either directly, or through the scoring interface -- the dropdown will include current repertory.). Downloading closer to the contest will change nothing.  Moreover, I do think I'm going to decouple these systems entirely, meaning that locking in the future will become a reality for both BS and legacy.    

TL;DR:  I think this misplaced.  I again think you'll want to change the language that threatens disqualification when that is not a credible threat.  And perhaps you want to let the CAs know they can check a group in  BS if a song is missing for the actual title.  But at the very least the session_verify_report text needs to change because it's inaccurate.

**Comment by millsjdm** [comment]
got it - I'm on it.

*J. David Mills*
SCJC Chairman 2016-2019
*Mobile Phone - 813.786.4381*

*Singing is Life is Singing is…*
